core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -49,10 +49,20 @@ class KryoSerializer(conf: SparkConf)
   with Logging
   with Serializable {
 
-  private val bufferSize =
-    (conf.getDouble("spark.kryoserializer.buffer.mb", 0.064) * 1024 * 1024).toInt
+  private val bufferSizeMb: Double = conf.getDouble("spark.kryoserializer.buffer.mb", 0.064)
```

Comment:
Declaring the type here *is* more descriptive, but I'd leave it out to be consistent with the other variables in the class.